### PR TITLE
image: build on PR as well

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
only publish once it is merged

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
+RUN dnf install git xorg-x11-server-Xvfb procps-ng wget libjpeg-turbo-devel expat-devel 'dnf-command(builddep)' -y
 RUN dnf builddep gtk4 -y
 
 # build gtk4 from the latest release
-ADD https://download.gnome.org/sources/gtk/4.6/gtk-4.6.0.tar.xz /tmp/gtk-4.6.0.tar.xz
-RUN tar -xf /tmp/gtk-4.6.0.tar.xz --directory /tmp
-WORKDIR /tmp/gtk-4.6.0
+#ADD https://download.gnome.org/sources/gtk/4.6/gtk-4.6.0.tar.xz /tmp/gtk-4.6.0.tar.xz
+#RUN tar -xf /tmp/gtk-4.6.0.tar.xz --directory /tmp
+# WORKDIR /tmp/gtk-4.6.0
```
Comment (Hofer-Julian) [inline Dockerfile:10]: Was it intended to merge these comments?

Comment (bilelmoussaoui) [reply Dockerfile:10]: yes

Comment (bilelmoussaoui) [reply Dockerfile:10]: I plan to switch again to 4.8.0 once it is released

Comment (Hofer-Julian) [reply Dockerfile:10]: Okay :)

**Pull request merged**